app/styles/partials/_info.scss

```diff
@@ -248,4 +248,58 @@ $packages: (
     width: 30px;
     order: 1;
   }
-}
\ No newline at end of file
+}
+
+.contributors img {
```

Comment:
In general, it's a good habit to order css declaration in alphabetical order. If you get used to that approach, it's much easier later to find specific property you need

Reply:
changed the order :+1: